torch/_dynamo/variables/builder.py

```diff
@@ -1506,7 +1506,7 @@ def wrap_tensor(self, value: torch.Tensor):
 
         self.tx.output.input_source_to_var[source] = tensor_variable
         assert "tensor_dict" not in tensor_proxy.node.meta
-        tensor_proxy.node.meta["tensor_dict"] = value.__dict__.copy()
+        tensor_proxy.node.meta["tensor_dict"] = {k: None if isinstance(v, torch.Tensor) else v for k, v in value.__dict__.items()}
```

Comment:
you should extract this into a utility function, `tensor_dict` is populated in multiple places